src/player.js

```diff
@@ -1849,8 +1847,10 @@ export default class Player extends FakeEventTarget {
    */
   _updateTracks(tracks: Array<Track>): void {
     Player._logger.debug('Tracks changed', tracks);
-    this._tracks = [...tracks, ...this._externalCaptionsHandler.getExternalTracks(tracks)];
+    this._tracks = tracks.concat(this._externalCaptionsHandler.getExternalTracks(tracks));
```

Comment:
what is the difference?

Reply:
Do you really prefer the previous??

Reply:
No preference, but the previous commit made this change, and now you are reverting it.
If this is just opinionated change please consult with whomever made the original change or at least explain why you reverted the code.
Is this pref related?

Reply:
The original was only `this._tracks = tracks`. while the external captions change it got to the red line above, and i'm changing the green line above. same logic but readable.